README.md

```diff
@@ -17,6 +19,18 @@ npm start
 ```
 The command prompt `KnowSheet> ` will appear.
 
+... or evaluate one expression from the command-line:
+```bash
+echo '(expression-here)' | npm start
+```
+
+... and use the result in another command:
```

Comment:
It was written `or` first :) then I thought that this line is also "evaluate one expression from the command-line".